2021-09-05 18:51:01,202 INFO    MainThread:24150 [wandb_setup.py:_flush():69] setting env: {}
2021-09-05 18:51:01,202 INFO    MainThread:24150 [wandb_setup.py:_flush():69] setting login settings: {}
2021-09-05 18:51:01,202 INFO    MainThread:24150 [wandb_init.py:_log_setup():342] Logging user logs to /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch/NLP/Disaster-Tweets/wandb/run-20210905_185101-1tfm66p1/logs/debug.log
2021-09-05 18:51:01,202 INFO    MainThread:24150 [wandb_init.py:_log_setup():343] Logging internal logs to /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch/NLP/Disaster-Tweets/wandb/run-20210905_185101-1tfm66p1/logs/debug-internal.log
2021-09-05 18:51:01,203 INFO    MainThread:24150 [wandb_init.py:_jupyter_setup():294] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7fdb766326d0>
2021-09-05 18:51:01,203 INFO    MainThread:24150 [wandb_init.py:init():375] calling init triggers
2021-09-05 18:51:01,203 INFO    MainThread:24150 [wandb_init.py:init():382] wandb.init called with sweep_config: {}
config: {}
2021-09-05 18:51:01,204 INFO    MainThread:24150 [wandb_init.py:init():424] starting backend
2021-09-05 18:51:01,204 INFO    MainThread:24150 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-09-05 18:51:01,234 INFO    MainThread:24150 [backend.py:ensure_launched():135] starting backend process...
2021-09-05 18:51:01,266 INFO    MainThread:24150 [backend.py:ensure_launched():140] started backend process with pid: 25067
2021-09-05 18:51:01,267 INFO    MainThread:24150 [wandb_init.py:init():429] backend started and connected
2021-09-05 18:51:01,314 INFO    MainThread:24150 [wandb_run.py:_label_probe_notebook():815] probe notebook
2021-09-05 18:51:01,315 INFO    MainThread:24150 [wandb_init.py:init():477] updated telemetry
2021-09-05 18:51:01,315 INFO    MainThread:24150 [wandb_init.py:init():500] communicating current version
2021-09-05 18:51:02,255 INFO    MainThread:24150 [wandb_init.py:init():505] got version response 
2021-09-05 18:51:02,256 INFO    MainThread:24150 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-09-05 18:51:02,779 INFO    MainThread:24150 [wandb_init.py:init():540] starting run threads in backend
2021-09-05 18:51:07,785 INFO    MainThread:24150 [wandb_run.py:_console_start():1601] atexit reg
2021-09-05 18:51:07,786 INFO    MainThread:24150 [wandb_run.py:_redirect():1475] redirect: SettingsConsole.WRAP
2021-09-05 18:51:07,786 INFO    MainThread:24150 [wandb_run.py:_redirect():1512] Wrapping output streams.
2021-09-05 18:51:07,786 INFO    MainThread:24150 [wandb_run.py:_redirect():1536] Redirects installed.
2021-09-05 18:51:07,787 INFO    MainThread:24150 [wandb_init.py:init():565] run started, returning control to user process
2021-09-05 18:51:07,787 INFO    MainThread:24150 [wandb_watch.py:watch():43] Watching
2021-09-05 18:51:46,858 INFO    MainThread:24150 [wandb_init.py:_pause_backend():256] pausing backend
2021-09-05 18:51:46,858 INFO    MainThread:24150 [jupyter.py:save_ipynb():301] looking for notebook: 00.ipynb
2021-09-05 18:52:09,641 INFO    MainThread:24150 [wandb_run.py:_atexit_cleanup():1571] got exitcode: 0
2021-09-05 18:52:09,642 INFO    MainThread:24150 [wandb_run.py:_restore():1543] restore
2021-09-05 18:52:09,882 INFO    MainThread:24150 [wandb_run.py:_restore():1543] restore
